clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -150,19 +209,19 @@ public void testStartupAndTearDown() throws InterruptedException {
     }
 
     @Test
-    public void testApplicationEvent() {
-        ApplicationEvent e = new PollEvent(100);
-        applicationEventsQueue.add(e);
+    void testRequestManagersArePolledOnce() {
         consumerNetworkThread.runOnce();
-        verify(applicationEventProcessor, times(1)).process(e);
+        requestManagers.entries().forEach(rmo -> rmo.ifPresent(rm -> verify(rm).poll(anyLong())));
```

Comment:
I think we're missing the expectations for the requestManagers.entries() to make sure there are some managers and we actually verify something, so we need  `when(requestManagers.entries()).thenReturn(..` with some managers.